### ISSUE TITLE
20190323 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#38]
# 20190323 Github 前端趋势榜
[Github 前端开发日报](http://caibaojian.com/c/news)栏目数据来自[Github热榜](http://news.caibaojian.com/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [clone95/Machine-Learning-Study-Path-March-2019](https://github.com/clone95/Machine-Learning-Study-Path-March-2019) （A complete ML study path, focused on TensorFlow and Scikit-Learn）
* [vuejs/vue](https://github.com/vuejs/vue) （        A progressive, incrementally-adoptable JavaScript framework for building UI on the web.      ）
* [alexfoxy/laxxx](https://github.com/alexfoxy/laxxx) （Simple &amp; light weight vanilla javascript plugin to create beautiful animations things when you scrolllll!! Harness the power of the most intuitive interaction and make your websites come alive!）
* [lyricat/wechat-format](https://github.com/lyricat/wechat-format) （微信公众号排版编辑器，Markdown 到给微信特制的 HTML）
* [scutan90/DeepLearning-500-questions](https://github.com/scutan90/DeepLearning-500-questions) （深度学习500问，以问答形式对常用的概率知识、线性代数、机器学习、深度学习、计算机视觉等热点问题进行阐述，以帮助自己及有需要的读者。 全书分为15个章节，近20万字。由于水平有限，书中不妥之处恳请广大读者批评指正。 未完待续............ 如有意合作，联系scutjy2015@163.com 版权所有，违权必究 Tan 2018.06）
* [CyC2018/CS-Notes](https://github.com/CyC2018/CS-Notes) （? Computer Science Learning Notes）
* [syhyz1990/baiduyun](https://github.com/syhyz1990/baiduyun) （油猴脚本 直接下载百度网盘和百度网盘分享的文件,直链下载超级加速）
* [nid666/PiracyArchive](https://github.com/nid666/PiracyArchive) （A complete backup of the Reddit r/Piracy sub）
* [facebook/react](https://github.com/facebook/react) （A declarative, efficient, and flexible JavaScript library for building user interfaces.）
* [Authman2/Mosaic](https://github.com/Authman2/Mosaic) （&#x1f3a8; A front-end JavaScript library for building user interfaces.）
* [Coalfire-Research/npk](https://github.com/Coalfire-Research/npk) （A mostly-serverless distributed hash cracking platform）
* [ammonb/stereogram-raycaster](https://github.com/ammonb/stereogram-raycaster) （A raycaster that renders to random dot stereogram）
* [C451/trading-vue-js](https://github.com/C451/trading-vue-js) （&#x1f4b9; Hackable charting lib for traders. You can draw literally ANYTHING on top of candlestick charts.）
* [LisaDziuba/Awesome-Design-Tools](https://github.com/LisaDziuba/Awesome-Design-Tools) （The best design tools for everything.）
* [mozilla/send](https://github.com/mozilla/send) （File Sharing Experiment）
* [agalwood/Motrix](https://github.com/agalwood/Motrix) （A full-featured download manger.）
* [airbnb/javascript](https://github.com/airbnb/javascript) （        JavaScript Style Guide      ）
* [30-seconds/30-seconds-of-code](https://github.com/30-seconds/30-seconds-of-code) （Curated collection of useful Javascript snippets that you can understand in 30 seconds or less.）
* [vvo/gifify](https://github.com/vvo/gifify) （&#x1f63b; Convert any video file to an optimized animated GIF.）
* [trekhleb/javascript-algorithms](https://github.com/trekhleb/javascript-algorithms) （Algorithms and data structures implemented in JavaScript with explanations and links to further readings）
* [GoogleChrome/puppeteer](https://github.com/GoogleChrome/puppeteer) （Headless Chrome Node API）
* [victordibia/handtrack.js](https://github.com/victordibia/handtrack.js) （A library for prototyping realtime hand detection (bounding box), directly in the browser.）
* [gatsbyjs/gatsby](https://github.com/gatsbyjs/gatsby) （⚛️?? Blazing fast static site generator for React）
* [axios/axios](https://github.com/axios/axios) （Promise based HTTP client for the browser and node.js）
* [NervJS/taro](https://github.com/NervJS/taro) （京东多端统一开发框架，支持用 React 的开发方式编写一次代码，生成能运行在微信小程序、H5、React Native 等的应用。）
* [alibaba/ice](https://github.com/alibaba/ice) （基于 React 的海量可复用区块，极速构建中后台应用）
* [zeit/next.js](https://github.com/zeit/next.js) （        Framework for server-rendered React apps      ）
* [facebook/create-react-app](https://github.com/facebook/create-react-app) （Create React apps with no build configuration.）
* [iview/iview-admin](https://github.com/iview/iview-admin) （Vue 2.0 admin management system template based on iView）

## Vue
* [dcloudio/uni-app](https://github.com/dcloudio/uni-app) （使用 Vue.js 开发跨平台应用的前端框架）
* [PanJiaChen/vue-element-admin](https://github.com/PanJiaChen/vue-element-admin) （        vue2 admin / a management system template      ）
* [ElemeFE/element](https://github.com/ElemeFE/element) （        A Vue.js 2.0 UI Toolkit for Web      ）
* [teamprevue/PreVue](https://github.com/teamprevue/PreVue) （&#x1f3a8; Single page application prototyping tool for developers and designers.）
* [iview/iview](https://github.com/iview/iview) （        A high quality UI Toolkit with both Vue.js 2.0 and 1.0      ）
* [buefy/buefy](https://github.com/buefy/buefy) （Lightweight UI components for Vue.js based on Bulma）
* [sendya/ant-design-pro-vue](https://github.com/sendya/ant-design-pro-vue) （??‍???‍? Use Ant Design Vue like a Pro! Demo: https://pro.loacg.com | React: http://pro.ant.design/）
* [heyui/heyui](https://github.com/heyui/heyui) （UI Toolkit for Web, Vue2.0）
* [salomonelli/best-resume-ever](https://github.com/salomonelli/best-resume-ever) （? ? Build fast ? and easy multiple beautiful resumes and create your best CV ever! Made with Vue and LESS.）
* [macrozheng/mall-admin-web](https://github.com/macrozheng/mall-admin-web) （mall-admin-web是一个电商后台管理系统的前端项目，基于Vue+Element实现。 主要包括商品管理、订单管理、会员管理、促销管理、运营管理、内容管理、统计报表、财务管理、权限管理、设置等功能。）
* [vueComponent/ant-design-vue](https://github.com/vueComponent/ant-design-vue) （An enterprise-class UI components based on Ant Design and Vue. ?）
* [didi/mand-mobile](https://github.com/didi/mand-mobile) （? A mobile UI toolkit, based on Vue.js 2, designed for financial scenes.）
* [liuzhuoling2011/baidupcs-web](https://github.com/liuzhuoling2011/baidupcs-web) （基于vue2+nuxt构建的高仿饿了么（2018版））
* [cossssmin/gridsome-starter-bleda](https://github.com/cossssmin/gridsome-starter-bleda) （Gridsome blog starter, built with Tailwind CSS）

## HTML
* [changtuiqie/WeChatAgreement.WebApi.Simple](https://github.com/changtuiqie/WeChatAgreement.WebApi.Simple) （mac微信,ipad微信 webapi封装好的实现方案，免IIS一键部署。 可实现微信80%功能；支持62数据登录、扫码登录、收发朋友圈、查看朋友圈、微信建群、微信拉人进群、微信公众号阅读、微信消息收发、微信附近的人定位、微信添加好友、微信红包接收、微信防撤回、分享小程序、微信加粉、微信收藏、微信标签等）
* [Alivon/Panda-Learning](https://github.com/Alivon/Panda-Learning) （熊猫学习Panda-Learning是一个辅助学习强国的软体,帮助挤不出时间，却仍然需要学习的积极分子，盼达学习强国(xuexiqiangguo)）
* [everitoken/evtjs](https://github.com/everitoken/evtjs) （API Bindings (SDK) for the everiToken blockchain.）
* [hamukazu/lets-get-arrested](https://github.com/hamukazu/lets-get-arrested) （This project is intended to protest against the police in Japan）
* [kubernetes/website](https://github.com/kubernetes/website) （Kubernetes website and documentation repo:）
* [google/styleguide](https://github.com/google/styleguide) （        Style guides for Google-originated open-source projects      ）
* [ghosh/micromodal](https://github.com/ghosh/micromodal) （⭕ Tiny javascript library for creating accessible modal dialogs）
* [swagger-api/swagger-codegen](https://github.com/swagger-api/swagger-codegen) （        swagger-codegen contains a template-driven engine to generate documentation, API clients and server stubs in different languages by parsing your OpenAPI / Swagger definition.      ）
* [portainer/portainer](https://github.com/portainer/portainer) （        Simple management UI for Docker      ）
* [iliakan/javascript-tutorial-en](https://github.com/iliakan/javascript-tutorial-en) （        Modern JavaScript Tutorial       ）
* [BlackrockDigital/startbootstrap-sb-admin-2](https://github.com/BlackrockDigital/startbootstrap-sb-admin-2) （        A free, open source, Bootstrap admin theme created by Start Bootstrap      ）
* [techqueria/website](https://github.com/techqueria/website) （&#x1f32e; We're a community of Latinx professionals in the tech industry.）
* [gcushen/hugo-academic](https://github.com/gcushen/hugo-academic) （        The personal website framework for Hugo. Demo at      ）
* [facebookresearch/fastText](https://github.com/facebookresearch/fastText) （Library for fast text representation and classification.）
* [uikit/uikit](https://github.com/uikit/uikit) （A lightweight and modular front-end framework for developing fast and powerful web interfaces）
* [interplanaria/bottle](https://github.com/interplanaria/bottle) （Bitcoin Browser）
* [dixonandmoe/rellax](https://github.com/dixonandmoe/rellax) （        Lightweight, vanilla javascript parallax library      ）
* [greggman/webgl-fundamentals](https://github.com/greggman/webgl-fundamentals) （WebGL lessons that start with the basics）
* [huyaocode/webKnowledge](https://github.com/huyaocode/webKnowledge) （前端知识点总结）

## CSS
* [twbs/bootstrap](https://github.com/twbs/bootstrap) （The most popular HTML, CSS, and JavaScript framework for developing responsive, mobile first projects on the web.      ）
* [daneden/animate.css](https://github.com/daneden/animate.css) （        A cross-browser library of CSS animations. As easy to use as an easy thing.      ）
* [github/personal-website](https://github.com/github/personal-website) （Code that'll help you kickstart a personal website that showcases your work as a software developer.）
* [tailwindcss/tailwindcss](https://github.com/tailwindcss/tailwindcss) （A utility-first CSS framework for rapid UI development.）
* [jgthms/bulma](https://github.com/jgthms/bulma) （        Modern CSS framework based on Flexbox      ）
* [carloscuesta/gitmoji](https://github.com/carloscuesta/gitmoji) （        An emoji guide for your commit messages. ?       ）
* [theme-next/hexo-theme-next](https://github.com/theme-next/hexo-theme-next) （Elegant and powerful theme for Hexo.）
* [necolas/normalize.css](https://github.com/necolas/normalize.css) （A collection of HTML element and attribute style-normalizations      ）
* [rstacruz/cheatsheets](https://github.com/rstacruz/cheatsheets) （My cheatsheets）
* [yueshutong/JavaMonitor](https://github.com/yueshutong/JavaMonitor) （一款适用于Java应用的在线性能监控系统（WY））
* [nostalgic-css/NES.css](https://github.com/nostalgic-css/NES.css) （NES-style CSS Framework | ファミコン風CSSフレームワーク）
* [Autodesk/standard-surface](https://github.com/Autodesk/standard-surface) （White paper describing Arnold's standard surface shader.）
* [Microsoft/TypeScript-Node-Starter](https://github.com/Microsoft/TypeScript-Node-Starter) （        A starter template for TypeScript and Node with a detailed README describing how to use the two together.      ）
* [the-paperless-project/paperless](https://github.com/the-paperless-project/paperless) （Scan, index, and archive all of your paper documents）
* [twbs/rfs](https://github.com/twbs/rfs) （Automated responsive font sizes）
* [Grsmto/simplebar](https://github.com/Grsmto/simplebar) （Custom scrollbars vanilla javascript library with native scroll, done simple, lightweight, easy to use and cross-browser.）
* [nikitavoloboev/my-mac-os](https://github.com/nikitavoloboev/my-mac-os) （? A list of applications, alfred workflows and various tools that make my macOS experience even more amazing）
* [iissnan/hexo-theme-next](https://github.com/iissnan/hexo-theme-next) （        Elegant theme for Hexo.       ）
* [picturepan2/spectre](https://github.com/picturepan2/spectre) （        Spectre.css - a lightweight, responsive and modern CSS framework.      ）
* [primer/css](https://github.com/primer/css) （The CSS design system that powers GitHub）
* [mmistakes/minimal-mistakes](https://github.com/mmistakes/minimal-mistakes) （        ? A flexible two-column Jekyll theme. Perfect for personal sites, blogs, and portfolios hosted on GitHub or your own server.      ）
* [Requarks/wiki](https://github.com/Requarks/wiki) （Wiki.js | A modern, lightweight and powerful wiki app built on NodeJS, Git and Markdown）

## TypeScript
* [codercom/code-server](https://github.com/codercom/code-server) （Run VS Code remotely.）

## Unknow
* [h5bp/Front-end-Developer-Interview-Questions](https://github.com/h5bp/Front-end-Developer-Interview-Questions) （        A list of helpful front-end related questions you can use to interview potential candidates, test yourself or completely ignore.      ）
* [ripienaar/free-for-dev](https://github.com/ripienaar/free-for-dev) （        A list of SaaS, PaaS and IaaS offerings that have free tiers of interest to devops and infradev      ）

## 关于我
 [前端开发博客](http://caibaojian.com/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](http://t.cn/EV7pcM1)。
如果你觉得我的分享对你有帮助，不妨考虑打赏我[一杯咖啡](http://caibaojian.com/donation)，以资鼓励。
![打赏](https://upload-images.jianshu.io/upload_images/570843-db4053c67a8c9ea9.png)
## 推荐
1. [掘金小册8折优惠，还有全网优惠教程](http://caibaojian.com/goods)
2. [36个小而美的赚钱项目，让你月入10万](http://t.cn/EV7YgYw)
## 微信群
加入微信群，每天晚上我会汇总一天最好的文章给你。
![qianduanquan](https://user-images.githubusercontent.com/3055447/38468989-651132ac-3b80-11e8-8e6b-15122322a9d7.png)
